sched/worker/noop.go

```diff
@@ -0,0 +1,27 @@
+package worker
+
+import (
+	"github.com/scootdev/scoot/cloud/cluster"
+	"github.com/scootdev/scoot/sched"
+)
+
+type noopWorkerController struct{}
+
+func (c *noopWorkerController) RunAndWait(task sched.TaskDefinition) error {
+	return nil
```

Comment:
Sure, how's this: now it's in a separate class, so we can use it, or not.

(And I added it to the worker demo. I had to up it from micro second to millisecond to make it noticeable)